pattern_library/monkey_utils.py

```diff
@@ -31,15 +31,15 @@ def node_render(context):
 
                 # Load pattern's config
                 current_template_name = parser.origin.template_name
```

Comment:
Suggestion: defining this variable here doesn't make much sense now we're not using to get the config. Perhaps remove this and replace the reference on L86 with `parser.origin.template_name` directly?